### PR TITLE
Fix multipart requests generation

### PR DESCRIPTION
**Pull request by StarProxima**
Solves https://github.com/Carapacik/swagger_parser/issues/144

### CODE REVIEW

File: .vscode/launch.json
Comment (Carapacik) [inline .vscode/launch.json:1]: Remove this file

File: swagger_parser/lib/src/utils/utils.dart [hunk 1]
```diff
@@ -90,7 +90,7 @@ const _generatedCodeComment = '''
 ''';
 
 const _ignoreLintsComment = '''
-// ignore_for_file: type=lint
+// ignore_for_file: type=lint, unused_import
```
Comment (Carapacik) [inline swagger_parser/lib/src/utils/utils.dart:93]: Why unused import?

File: swagger_parser/swagger_parser.yaml
Comment (Carapacik) [inline swagger_parser/swagger_parser.yaml:1]: Delete this file, there will be an error when running the program

**Review by Carapacik — CHANGES_REQUESTED** [review]

File: .vscode/launch.json
Comment (StarProxima) [reply .vscode/launch.json:1]: This is the startup config file for vscode, it is usually added to the git. But ok, I'll remove it.

File: swagger_parser/swagger_parser.yaml
Comment (StarProxima) [reply swagger_parser/swagger_parser.yaml:1]: The file itself is bypassed to run the parser in debug mode. If there is another way to run the parser, please write a small instruction how to do it.

Comment (Carapacik) [reply swagger_parser/swagger_parser.yaml:1]: This file is taken as the default if pulled from pub dev, so delete it.

**Review by Carapacik — APPROVED** [review]

**Pull request merged**